src/frontend/src/lib/types/errors.ts

```diff
@@ -1 +1,13 @@
+import type { TokenId } from '$lib/types/token';
+
 export class UserProfileNotFoundError extends Error {}
+
+export class AddressError extends Error {
```

Comment:
Should we call this `LoadIdbAddressError`?

Reply:
yes! better eheh!